storage/store/consumer_progress_test.go

```diff
@@ -0,0 +1,58 @@
+package store
```

Comment:
This is actually renamed from storage/badger/consumer_progress_test.go, github wasn't able to recognize this move, but think it's a new file.